lib/galaxy/webapps/galaxy/services/history_contents.py

```diff
@@ -483,6 +483,8 @@ def get_dataset_collection_archive_for_download(self, trans, id: DecodedDatabase
         try:
             dataset_collection_instance = self.__get_accessible_collection(trans, id)
             return self.__stream_dataset_collection(trans, dataset_collection_instance)
+        except (exceptions.ItemAccessibilityException, exceptions.RequestParameterInvalidException):
```

Comment:
I think we can drop the try/except completely, I don't see what this would help with.

Reply:
I assumed the exception logging below could be useful for some archiving errors and similar. We can drop it all if we don't care about that.

Reply:
I don't think that's done. See also https://github.com/galaxyproject/galaxy/issues/14899 -- we need to get rid of this throughout the codebase IMO.

Reply:
Sounds good. Done in [edb0e1e](https://github.com/galaxyproject/galaxy/pull/14898/commits/edb0e1ecfd9ba1f3f54176819bf1d3813fa284e2)

Reply:
No, I mean who would parse the logs for stuff like `Error in API while creating dataset collection archive`, that is so custom I can't believe you can come up with analytics for that.